pkg/networkmanager/interfaces.js

```diff
@@ -3751,7 +3751,7 @@ PageNetworkBondSettings.prototype = {
             interface_name: self.settings.bond.interface_name,
             assigned_mac_address: mac,
             monitoring_interval: options.miimon || options.arp_interval || "100",
```

Comment:
if ARP has now precedence over MII, should this one be flipped as well?

Reply:
MII has priority over ARP. `Use `options.arp_interval` instead, as when neither is set up `Mii` is preferred `